analytics_data_api/v0/views/learners.py

```diff
@@ -60,6 +60,7 @@ class LearnerView(LastUpdateMixin, CourseViewMixin, generics.RetrieveAPIView):
         Returns metadata and engagement data for the learner in JSON format.
 
             * username: The username of the enrolled learner.
+            * account_url:  URL to learner's account page.
```

Comment:
I can see how that would be valuable as well!

We will need the API endpoint as well in order to eventually fetch the profile image, bio, and other fields.